java/src/com/ibm/streamsx/topology/internal/context/remote/BuildServiceRemoteRESTWrapper.java

```diff
@@ -103,7 +105,10 @@ void remoteBuildAndSubmit(JsonObject submission, File archive) throws ClientProt
 			JsonObject response = doSubmitJobFromBuildArtifactPut(httpclient, deploy, apiKey, artifactId);
 			
 			// Pass back to Python
-			submission.add(RemoteContext.SUBMISSION_RESULTS, response);
+			final JsonObject submissionResult = GsonUtilities.getProperty(submission, RemoteContext.SUBMISSION_RESULTS);
+            for (Entry<String, JsonElement> entry : response.entrySet()) {
```

Comment:
Seems like this would be a useful utility to add to GsonUtilities since you use it twice. Add all in properties in one object to another.